src/google/protobuf/any.proto

```diff
@@ -95,7 +95,7 @@ option objc_class_prefix = "GPB";
 //
 //
 // JSON
-// ====
+//
```

Comment:
Can you use some other separator instead of totally removing the separator? e.g. #

Reply:
> Can you use some other separator instead of totally removing the separator? e.g. #

Not as far as I know - godoc doesn't use an explicit markup for headings - but "godoc recognises a line that starts with a capital letter and does not end in a full stop as a heading."

btw You can see the issue that this commit is fixing [here](https://pkg.go.dev/google.golang.org/protobuf/types/known/anypb)

  - "Example 3" and "Example 4" are nested inside "Example 2"
  - the JSON heading is interpreted as a paragraph and rendered as `JSON ==== The JSON representation of an `Any` value...`